src/main/java/org/assertj/core/presentation/CompositeRepresentation.java

```diff
@@ -0,0 +1,32 @@
+package org.assertj.core.presentation;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+import static org.assertj.core.presentation.StandardRepresentation.STANDARD_REPRESENTATION;
+
+public class CompositeRepresentation implements Representation {
+  protected final Map<String, Representation> representationsByName;
+
+  public CompositeRepresentation(Map<String, Representation> representations) {
+    if (representations == null) throw new IllegalArgumentException();
+    this.representationsByName = representations;
+  }
+
+  @Override
+  public String toStringOf(Object object) {
+    return (new ArrayList<Representation>(this.representationsByName.values()))
+      .stream().map(x -> x.toStringOf(object)).findAny().orElse(STANDARD_REPRESENTATION.toStringOf(object));
```

Comment:
this is going to change when we use priority, also this was not really following https://github.com/assertj/assertj-core/issues/2048#issuecomment-733214647 since there is no check that the representation returned null (meaning that it did not represented the value).

Reply:
@joel-costigliola I am not quite sure what should I return if the representation returns null. According to the comment in #2048, I saw the null will be returned if the representation returns null by default. So could you give me more details so that I can modify the code? Thanks!😁

Reply:
In our case null means the representation did not have a specific representation for the object, this is true except for the standard representation (at some point we need to return a representation for null).
This means we need to return the first non null values from the custom representations, if all of them returns null then we use the standard representation.
Does that make sense?

Reply:
Oh, yes! I understand. Thank you very much!